### PR TITLE
feat: Issuer API integration with Profile for multi-tenancy

### LINKED ISSUE #176: Issuer API - Support for multitenancy

### PR DESCRIPTION
**Pull request by rolsonquadras**
closes #176 

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: pkg/restapi/vc/operation/operations.go [hunk 1]
```diff
@@ -773,32 +774,46 @@ func (o *Operation) writeErrorResponse(rw http.ResponseWriter, status int, msg s
...
 	}
 
-	// use issuer id if options doesn't specify the DID
-	signerDID := validatedCred.Issuer.ID
-	if cred.Opts != nil && cred.Opts.AssertionMethod != "" {
-		signerDID = cred.Opts.AssertionMethod
+	// update the signing profile with the request options
+	err = updateIssueCredSigningProfile(o.vdri, &cred, profile)
+	if err != nil {
+		o.writeErrorResponse(rw, http.StatusBadRequest, fmt.Sprintf("failed to update signing profile:"+
```
Comment (sudeshrshetty) [inline pkg/restapi/vc/operation/operations.go:809]: http error code should be internal server error,

Comment (rolsonquadras) [reply pkg/restapi/vc/operation/operations.go:809]: In this case, we are updating the profile based on request (&cred is a request). Hence added bad-request.

Comment (sudeshrshetty) [reply pkg/restapi/vc/operation/operations.go:809]: I saw call to vdri there, that's why.

Comment (rolsonquadras) [reply pkg/restapi/vc/operation/operations.go:809]: yeah but we are using VDRI to resolve user provided DID.

**Pull request merged**